app/src/main/assets/web_extensions/coil/static/goSettings.js

```diff
@@ -0,0 +1,2 @@
+const url = 'https:/coil.com/settings'
```

Comment:
Shouldn't that be `http://` (note, two forward slashes)?

Reply:
wow good catch - i will forward to that team

Reply:
https://github.com/coilhq/web-monetization-projects/blob/master/packages/coil-extension/static/goSettings.js was the original location - am advising them